stroyprombeton/request_data.py

```diff
@@ -52,10 +40,10 @@ class FetchProducts(Category):
 
     @property
     def id(self):
-        # @todo #443:15m Check FetchProducts post params.
-        #  Raise Http400 if categoryId not exists.
-        #  See `Http400` error on SE for example.
-        return self.request.POST.get('categoryId')
+        id_ = self.request.POST.get('categoryId')
+        if not id_:
+            raise Http400('POST param categoryId is missed')
```

Comment:
You should handle this exception in views, otherwise the server will produce 500 response

Reply:
it seems to me that not. So, i'll write test to check this moment

Reply:
seems we have no mech to autocast exceptions to bad request responses.
Fixed code and appended test